py/server/deephaven/stream/kafka/consumer.py

```diff
@@ -275,19 +278,44 @@ def _consume(
     except Exception as e:
         raise DHError(e, "failed to consume a Kafka stream.") from e
 
+class ProtobufProtocol(JObjectWrapper):
+    """The protobuf serialization / deserialization protocol."""
+
+    j_object_type = jpy.get_type("io.deephaven.kafka.protobuf.Protocol")
+
+    @staticmethod
+    def serdes():
```

Comment:
```suggestion
    def serdes() -> ProtobufProtocol:
```

Reply:
Unfortunately, Python throws an error when trying to reference this class.

Reply:
Try `def serdes() -> 'ProtobufProtocol':`